app/code/community/Reverb/ReverbSync/Helper/Data.php

```diff
@@ -33,6 +33,7 @@ public function createOrUpdateReverbListing($product, $do_not_allow_creation = f
     {
         // Create empty wrapper in the event an exception is thrown
         $listingWrapper = Mage::getModel('reverbSync/wrapper_listing');
+        /* @var $listingWrapper Reverb_ReverbSync_Model_Wrapper_Listing */
```

Comment:
/* @var $listingWrapper Reverb_ReverbSync_Model_Wrapper_Listing */

Comments such as this serve multiple purposes. One, it allows someone reviewing the code to know which classes the code intended to be using. Second, when using an IDE (such as PHPStorm which I use) comments such as these allow the IDE to verify that methods being called with object $listingWrapper not only exist, but are returning variables of the expected type (due to PHP comment docs). Third, if you can CTRL+click on "setMagentoProduct" in line 37 and the IDE will take you to the definition of the method.

These comments will help both in future development/refactoring of the model and for developers who are vetting out the code when deciding whether to include it on a Magento instance